.env

```diff
@@ -0,0 +1,3 @@
+ALCHEMY_RPC_URL="https://base-goerli.alchemyapi.io/v2/YOUR_ALCHEMY_KEY"
```

Comment:
This is an incorrect key, /YOUR_ALCHEMY_KEY? definetly an invalid key. Keys likes this would probably be gotten on the Alchemy dashboard